### PR TITLE
JMLL-410 - Cookie notice

### PR DESCRIPTION
**Pull request by jafacakes2011**

### CODE REVIEW

File: core/cookies/views.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from django.template.response import TemplateResponse
+
+
+def cookie_notice(request):
+    return TemplateResponse(request, "cookies/notice.html", {})
+
+
+def cookie_response(request: HttpRequest, response: Literal["accept", "reject"]):
+    redirect_response: HttpResponseRedirect = redirect(request.GET.get("next", "/"))
+    redirect_response.set_cookie(
```
Comment (jafacakes2011) [inline core/cookies/views.py:15]: @rossmiller it looks like this isn't stored for anonymous users.

File: leavers/templates/base.html [hunk 2]
```diff
@@ -2,167 +2,147 @@
...
-<script src="https://unpkg.com/dagre@0.7.4/dist/dagre.js"></script>
-<script src="/static/cytoscape-dagre/cytoscape-dagre.js"></script>
-{% block scripts %}{% endblock %}
-</body>
+        <script src="/static/govuk-frontend/govuk/all.js"></script>
+        <script src="/static/cytoscape/dist/cytoscape.min.js"></script>
+        <script src="https://unpkg.com/dagre@0.7.4/dist/dagre.js"></script>
+        <script src="/static/cytoscape-dagre/cytoscape-dagre.js"></script>
+        {% if COOKIE_RESPONSE == "accept" %}
+            {# Add GA script #}
```
Comment (jafacakes2011) [inline leavers/templates/base.html:144]: @rossmiller TODO: Add Google Analytics script here?

Comment (rossmiller) [reply leavers/templates/base.html:144]: Yes indeed. Merge anyway and add a ticket for it if you would be so kind?

File: core/cookies/views.py (hunk 1 above)
Comment (rossmiller) [reply core/cookies/views.py:15]: That's fine.

**Review by rossmiller — APPROVED** [review]
LGTM!

File: leavers/templates/base.html (hunk 2 above)
Comment (jafacakes2011) [reply leavers/templates/base.html:144]: Done! (See ticket in Jira)

**Pull request merged**